### ISSUE TITLE
`GitSavvy: check config`

### ISSUE THREAD

**Issue by divmain** [divmain/GitSavvy#340]
This command will check things like:
- which Git binary are you using?
- do you have git-flow installed?
- which version of Git are you using?

And report any problems that may occur with the configuration, like:
- `git push --force` behavior
- Cygwin git issues

**Comment by asfaltboy** [comment]
I think it can be a good idea to run the command on first install and show a warning alert to user if version/s are old/unsupported.

**Comment by stoivo** [comment]
Yeah, this would be nice to load commit template here too. So we don't need to read from disk si many times. 

https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration
look under: commit.template
